{
  setFailFast(true);
  try {
    eval(input);
    fail("Expected error '" + msg + "' but got no error");
  }
 catch (  IllegalArgumentException|EvalException e) {
    assertThat(e).hasMessage(msg);
  }
}
